cert-generation/webhooks/webhook.go

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright (c) 2019-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package webhooks
+
+import (
+	"github.com/devfile/devworkspace-operator/pkg/webhook/workspace"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// WebhookInit Initialize the webhook that denies everything until controller is started succesfully
+func WebhookInit(clientset *kubernetes.Clientset, namespace string) error {
+	configuration := workspace.BuildMutateWebhookCfg(namespace)
+
+	// Create mutating webhook
+	if _, err := clientset.AdmissionregistrationV1beta1().MutatingWebhookConfigurations().Get(configuration.Name, metav1.GetOptions{}); apierrors.IsNotFound(err) {
```

Comment:
It could be done with one operation - `create` and just check error if it's not already exist